packages/ui-kit/src/components/SimpleFilter/SimpleFilter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { SyntheticEvent, useRef } from 'react'
+import { useTopValues } from '../../hooks'
+import { FilterInput, FilterOperator, getTimeZone } from '../../helpers'
+
+import { Autocomplete } from '../Autocomplete'
+import { AutocompleteOption } from '../Autocomplete/Autocomplete.types'
+import { useFilter } from '../FilterProvider/useFilter'
+
+import { SimpleFilterProps } from './SimpleFilter.types'
+import { withContainer } from '../withContainer'
+import { ErrorFallback } from '../ErrorFallback'
+import { Loader } from '../Loader'
+import componentStyles from './SimpleFilter.module.scss'
+
+const SimpleFilterComponent = ({
+  autocompleteProps,
+  columnName: columnNameProp = '',
+  query,
+  errorFallbackProps,
+  error,
+  loading,
+  loaderProps,
+  options = []
+}: SimpleFilterProps) => {
+  const id = useRef(Symbol()).current
+
+  const isStatic = !query
+
+  const { filters, setFilters } = useFilter()
+
+  const columnName = query?.columnName ?? columnNameProp
+  const zone = query?.timeZone ?? getTimeZone()
+
+  const { data, error: hasError, isLoading } = useTopValues({ ...query, timeZone: zone })
+
+  const handleChange = (_: SyntheticEvent<Element, Event>, value: AutocompleteOption | null) => {
+    if (value == null) return
+
+    const newFilter: FilterInput = {
+      column: columnName,
+      operator: FilterOperator.Equals,
+      value: value.label
+    }
+
+    const newFilterList = filters.filter((filter) => filter.id !== id).concat({ ...newFilter, id })
+
+    setFilters([...newFilterList])
+  }
+
+  const autocompleteOptions = isStatic
+    ? options?.map((label) => ({ label })) ?? []
+    : data?.topValues.values.map((label) => ({ label })) ?? []
+
+  if (error != null || hasError != null) {
+    return <ErrorFallback error={null} {...errorFallbackProps} style={{ height: '34.2px' }} />
+  }
+
+  if (loading || (!isStatic && isLoading)) {
+    return (
+      <Loader {...loaderProps} className={componentStyles.loader} style={{ ...autocompleteProps?.containerStyle }} />
+    )
+  }
+
+  return <Autocomplete {...autocompleteProps} options={autocompleteOptions} onChange={handleChange} />
```

Comment:
Is the user allowed to type free-form text? For example, imagine they ask for top K values where K equals 10. But there's actually 100 values in the column. Are they still allowed to type out those other values, even if they don't show up in the list?

Reply:
If there is an error fetching the top K values, what do we do? Allow the user to just type free-form? (And how do we handle the error?)

Reply:
> Is the user allowed to type free-form text? For example, imagine they ask for top K values where K equals 10. But there's actually 100 values in the column. Are they still allowed to type out those other values, even if they don't show up in the list?

Currently no, but I think the case you mention is a good reason to support that, we can enable/disable free-solo via props

> If there is an error fetching the top K values, what do we do? Allow the user to just type free-form? (And how do we handle the error?)

Currently if the query fails I'm showing an error but I think it's a good idea to enable free-solo in this case instead and log an error